Mathlib/Lean/SourceInfo_fromRef.lean

```diff
@@ -0,0 +1,19 @@
+/-
+Copyright (c) 2022 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+
+import Lean
+
```

Comment:
This file should just be called `SourceInfo.lean`.

Reply:
Done